### PR TITLE
Adding UX designer job description

### PR DESCRIPTION
**Pull request by christinaforney**

### CODE REVIEW

File: job-descriptions/ux-designer.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+You will:
+
+- Own the comprehensive user experience of the product.
+- Contribute to overall strategy and decision-making about Sourcegraph’s features.
+- Work closely and effectively communicate with our product manager and engineers to define, design, and improve Sourcegraph, ensuring a high-quality implementation.
+- Support multiple projects simultaneously while meeting tight deadlines.
+- Create designs that adhere to and extend the existing style guide.
+
+### Ideal candidates have
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: Many of these seem like minimum qualifications (see SWE job description for how I separated qualifications from "nice to have") as opposed to ideal attributes.

File: job-descriptions/ux-designer.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- An understanding of and passion for what makes a great user experience.
+- A portfolio showcasing their skillset and years of experience designing digital products.
+- Experience (limited, or as a hobby, is fine) as a programmer to understand the product and user need, and to jump into our codebase and fix design (e.g. CSS) issues.
+- A knack for giving and receiving feedback, and explaining the reasoning behind their designs.
+- Experience using Figma, Sketch, or any other vector-based design tools to deliver product ideas from conception to mockups and finished renderings.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: Is this a qualification? It is hard to imagine a designer being successful without experience like this.

File: job-descriptions/ux-designer.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+1.  We give you a small design exercise, then set up a call for you to present your solution and discuss how you approached the problem.
+1.  We schedule a few more hours of interviews. We're happy to fly you out to our San Francisco office, or conduct the remaining interviews over video chat, whatever works best for you.
+1.  We check your references.
+1.  We make you a job offer.
+
+We also expect you to be interviewing us too, so ask us any questions you have along the way.
+
+If you aren't ready to start interviewing, but would like to stop by our San Francisco office to meet the team, let us know!
+
+**[Click here to apply]()**
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: I created a stub job on Google Hire.
```suggestion
**[Click here to apply](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAC5MKPtg8cFnBD?trackingTag=careersRepository)**
```

File: job-descriptions/ux-designer.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+---
+
+Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: This is already included in the about us section

**Comment by christinaforney** [comment]
> Thanks for getting started on this! I have some feedback from my own perspective, but I have never written a description for a designer before so I don't know what "speaks" best to this kind of candidate. (It would be great to get feedback from some designers and reference other designer job descriptions if you haven't done this already).

Thanks for all your input! I already reached out to a few designers I know and asked them to review, and hope to hear from them in the next few days.

File: job-descriptions/ux-designer.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are searching for a creative, self-motivated UX designer who can brainstorm ideas and bring them to life. We highly value Product Design within our organization, as it impacts all aspects of Sourcegraph’s product. You'll have creative autonomy and a strong influence over the product’s development.
+
+You will:
+
+- Own the comprehensive user experience of the product.
```
[the remark this answers is not in the document]
Comment (christinaforney) [reply job-descriptions/ux-designer.md]: I see what you mean and agree. What I was going for here is that they have ownership over making sure there is a cohesive experience when using the product. They would help the team to move the product towards that comprehensive vision. I'll reword!

File: job-descriptions/ux-designer.md (hunk 1 above)
Comment (christinaforney) [reply job-descriptions/ux-designer.md]: I'm refactoring to match your format - I like it!

**Comment by christinaforney** [comment]
@nicksnyder - made a lot of changes to refactor and improve wording. Let me know what you think!

File: job-descriptions/ux-designer.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+- Collaborate directly with customers to collect feature requirements and inform product design.
+- Publish blog posts and give conference talks about your work at Sourcegraph.
+
+#### Your qualifications
+
+- A passion for and ability to communicate what makes a great user experience.
+- Experience using Figma, Sketch, or any other vector-based design tools to deliver product ideas from conception to mockups and finished renderings.
+- A portfolio showcasing your systematic and creative design thinking as well as deep understainding of UI/UX best practices for digital products.
+- Experience (limited, or as a hobby, is fine) as a programmer to understand the product and user need, and to jump into our codebase and fix design (e.g. CSS) issues.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: Is this actually a "nice to have"?

File: job-descriptions/ux-designer.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+- A knack for giving and receiving feedback, and explaining the reasoning behind their designs.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]:
```suggestion
- A knack for giving and receiving feedback, and explaining the reasoning behind your designs.
```

This kind of feels duplicative with the first point. Possible to combine?

File: job-descriptions/ux-designer.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+- Help measure, learn, and build new and existing experiences.
+- Collaborate with product and engineers to bring ideas to life, and support the team to make incremental progress toward design excellence.
+- Leverage and evolve the style guide to create a clean, usable design system.
+
+We will encourage and support you to:
+
+- Collaborate directly with customers to collect feature requirements and inform product design.
+- Publish blog posts and give conference talks about your work at Sourcegraph.
+
+#### Your qualifications
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: This feels like a lot of qualifications. Is it possible to trim this list down into something more concise without losing important fidelity?

In any case, make sure that this list is stack ranked by importance (most important at top).

File: job-descriptions/ux-designer.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world and about half work from our office in San Francisco.
+
+### Compensation and benefits
+
+We provide competitive pay and equity because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](../README.md#benefits) to keep you happy, healthy, and productive.
+
+### Interview process
+
+1.  You [apply here]().
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]:
```suggestion
1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAC5MKPtg8cFnBD?trackingTag=careersRepository).
```

File: job-descriptions/ux-designer.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are searching for a creative, self-motivated UX designer who can iteratively improve the cohesive user expereience, while bringing clean and thoughtful UI to their designs. It is important that this person deeply understands developers and can craft an experience that delights a technical audience.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: > It is important that this person deeply understands developers

Is this a pre-requisite to applying?

Comment (christinaforney) [reply job-descriptions/ux-designer.md]: Adjusted role description and added this to the qualifications instead.

**Comment by christinaforney** [comment]
These are the last of the major refactors based on feedback from a few designers + @nicksnyder. Will do final review with a UX designer friend.

File: job-descriptions/ux-designer.md (hunk 6 above)
Comment (christinaforney) [reply job-descriptions/ux-designer.md]: Reworded.

File: job-descriptions/ux-designer.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+- Proficiency with Figma, Sketch, or similar tools to design end-to-end experiences (UX, interaction, and visual).
+- Experience with usability testing, customer journey mapping, developing personas, wireframing and prototyping, sketching, and other UX design methods.
+- The ability to adapt your tools and processes for the velocity of a fast-paced startup where projects have varying requirement levels.
+- The ability to proactively give maintenance to the design system when thinking about how new concepts impact the whole product.
+- A knack for giving and receiving feedback, and explaining the reasoning behind your designs.
+- Deep understanding of developers, and a good understanding of version control systems, preferably Git.
+
+#### Nice to haves
+
+- You have experience designing developer tools.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: Do you want to have a "You have:" prefix like the above section so you can remove it from this line item?

```suggestion
You have:

- Experience designing developer tools.
```

File: job-descriptions/ux-designer.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+- Experience working in cross-functional teams on multiple projects in tandem.
+- Experience building or contributing to the design of consumer or enterprise applications.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world and about half work from our office in San Francisco.
```
Comment (nicksnyder) [inline job-descriptions/ux-designer.md]: This specifically mentions our engineering team, which isn't particularly relevant to designers.

```suggestion
We are flexible and don't require you to be in any particular timezone or location. Our team is distributed across the world and less than half work from our office in San Francisco.
```

File: job-descriptions/ux-designer.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+- Contribute to the overall strategy and decision-making about Sourcegraph’s features.
+- Drive continual progress towards a cohesive, delightful experience for a technical audience through iterative improvements.
+- Have the autonomy to facilitate conversations, solve problems, and execute the designs.
+- Pull together insights to propose a variety of solutions, then collaborate with product and engineering to determine the best solutions to problems.
+- Use prototyping and usability testing to inform and validate designs.
+
+We will encourage and support you to:
+
+- Collaborate directly with customers to inform strategy and design decisions.
```
[the remark this answers is not in the document]
Comment (christinaforney) [reply job-descriptions/ux-designer.md]: Good point. Yes, I think we do want them to feel responsibility around interacting with customers to inform the design. Moving it up.

**Comment by christinaforney** [comment]
Got positive feedback on changes. Am going to publish and if any changes are needed follow up in another PR.

**Pull request merged**